tests/test_cnn/test_weight_init.py

```diff
@@ -47,6 +47,16 @@ def test_normal_init():
     # TODO: sanity check distribution, e.g. mean, std
 
 
+def test_truncated_normal_init():
+    conv_module = nn.Conv2d(3, 16, 3)
```

Comment:
Suggest merge this function with `test_trunc_normal_init()`

Reply:
got it